cuda_bindings/cuda/bindings/_internal/cufile_linux.pyx

```diff
@@ -97,24 +116,9 @@ cdef int _check_or_init_cufile() except -1 nogil:
         return 0
 
     cdef void* handle = NULL
-    cdef int err, driver_ver = 0
 
     with gil, __symbol_lock:
-        # Load driver to check version
-        handle = dlopen('libcuda.so.1', RTLD_NOW | RTLD_GLOBAL)
-        if handle == NULL:
-            err_msg = dlerror()
-            raise NotSupportedError(f'CUDA driver is not found ({err_msg.decode()})')
-        global __cuDriverGetVersion
-        if __cuDriverGetVersion == NULL:
-            __cuDriverGetVersion = dlsym(handle, "cuDriverGetVersion")
-        if __cuDriverGetVersion == NULL:
-            raise RuntimeError('something went wrong')
-        err = (<int (*)(int*) noexcept nogil>__cuDriverGetVersion)(&driver_ver)
-        if err != 0:
-            raise RuntimeError('something went wrong')
-        #dlclose(handle)
-        handle = NULL
+        driver_ver = get_cuda_version()
```

Comment:
`driver_ver` is unused in the `load_library` call which is the only place that it's passed as far as I can tell. Should we just remove it?

Reply:
Fine with me to remove.  I was just trying not to change existing behavior.  I thought (perhaps) there was a reason we always "check if CUDA is installed" before loading the specific library.  @leofang: Do you know if it matters?

Reply:
Explained and tracked in https://github.com/NVIDIA/cuda-python/issues/962